features/range.yml

```diff
@@ -0,0 +1,33 @@
+name: Range
+description: The Range interface represents a fragment of a document that can contain nodes and parts of text nodes.
```

Comment:
`Range` is defined in https://dom.spec.whatwg.org/#interface-range, but on its own it doesn't really do much, it's mostly in support of https://w3c.github.io/selection-api/.

I we may need a DOM feature for most old parts of https://dom.spec.whatwg.org/ that other specs depend on. @ddbeck WDYT?

Reply:
Until we're backed into a corner, I think we ought to put such interfaces with their most longstanding dependent features (e.g., unless someone has an older use-case, `Range` can go with the selection API—not sure what use `Range` would be alone).

That said, if there's no such feature or it has use alone—supposing I'm wrong about `Range` here—then I'd be OK with a foundational DOM feature. It might be quite boring, but make a lot of sense to stand in contrast with the Shadow DOM feature, with "Also known as light DOM" description text.